active_model_command.gemspec

```diff
@@ -28,5 +28,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "rspec_junit_formatter", "~> 0.4"
 
-  spec.add_runtime_dependency "activemodel", ">= 2.2.0"
+  spec.add_runtime_dependency "sqlite3", "~> 1.4"
+  spec.add_runtime_dependency "activerecord", ">= 2.2.0"
```

Comment:
We need active record in order to work with transactions. We won't need to require `activemodel` since `activerecord` [already depends on that](https://github.com/rails/rails/blob/main/activerecord/activerecord.gemspec#L38).